src/ui/ToggleBox.tsx

```diff
@@ -27,39 +25,33 @@ const ToggleBox = ({
     <Stack
       forwardedAs="button"
       onClick={onClick}
-      padding={5}
+      padding={5.5}
```

Comment:
I would keep using non-decimal numbers. Since we have only used normal ints as padding, margin and spacing. But if its really an obvious improvement like this, keep it 😉